pyfar/constants/constants.py

```diff
@@ -434,3 +435,115 @@ def density_of_air(
 
     # Equation 6.71
     return P / (R_a * (temperature + 273.15)) * (1+C)/(1+alpha*C)
+
+
+def octave_band_tolerance(
+        exact_center_frequency: float,
+        num_fractions: Literal[1, 3],
+        tolerance_class : Literal[1, 2]):
+    r"""
+    Calculate the tolerance limits for fractional octave band filters.
+
+    Calculation is in accordance with IEC 61260-1:2014 [#]_ (Section 5.10 and
+    Table 1).
+
+    .. note ::
+        The standard defines some lower tolerance limits to :math:`-\infty`,
+        which is inconvenient for plotting. The returned tolerance is -60000 dB
+        in these cases, which is below the smallest possible value approx.
+        -6000 dB (``20*np.log10(np.finfo(float).tiny)``).
+
+    Parameters
+    ----------
+    exact_center_frequency : float
+        The exact center frequency of the band filter in Hz (see
+        :py:func:`~pyfar.dsp.filter.fractional_octave_frequencies`).
+    num_fractions : Literal[1, 3]
+        The number of bands an octave is divided into. ``1`` for octave bands
+        and ``3`` for third octave bands.
+    tolerance_class : Literal[1, 2]
+        The tolerance class as defined in the standard. Must be 1 or 2.
```

Comment:
Please keep this consistent
```suggestion
        The tolerance class as defined in the standard. Must be ``1`` or ``2``.
```